ios/KlaviyoBridge.swift

```diff
@@ -3,210 +3,215 @@ import KlaviyoSwift
 
 @objc
 public class KlaviyoBridge: NSObject {
-  enum ProfileProperty: String, CaseIterable {
```

Comment:
I ran xcode's fix indent thingy on this file.

Reply:
😢

Reply:
so no code change?

Reply:
only the addition of the getPushToken method that i commented on below